### ISSUE TITLE
gnt-cluster masterfailover --no-voting is not automat-able

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#338]
Originally reported of Google Code with ID 282.

```
As discussed in https://groups.google.com/d/topic/ganeti/zzjtvPPAvDI/discussion.
```
Originally added on 2012-10-28 18:23:28 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
I've pushed a fix for this; it will be released as part of 2.6.2.
```
Originally added on 2012-10-29 12:23:13 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.6

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking as released the bugs fixed in 2.6.2.
```
Originally added on 2013-01-11 12:13:05 +0000 UTC.

Changed State: Released
